org.kde.isoimagewriter.json

```diff
@@ -0,0 +1,35 @@
+{
+   "id": "org.kde.isoimagewriter",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "5.15-22.08",
+   "sdk": "org.kde.Sdk",
+   "command": "isoimagewriter",
+   "finish-args": [
+      "--device=dri",
+      "--filesystem=host",
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--system-talk-name=org.freedesktop.UDisks2",
+      "--talk-name=org.freedesktop.Flatpak"
```

Comment:
is this access really necessary? The previous attempt #3031 didn't require it.

Access to this name require an exception for the linter.

Also about the filesystem access: https://github.com/flathub/flathub/pull/3031/files#r852934978

Reply:
The app now uses udisks2 so the udisks2 permission is necessary.  The previous attempt failed.  

Filesystem access is needed to read the ISO files to write.

Reply:
dunno, I guess not, removed